src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,28 @@
-export const PostInfo = () => <>Put the post here</>;
+import { CommentList } from '../CommentList/CommentList';
+import { UserInfo } from '../UserInfo/UserInfo';
+
+import './PostInfo.scss';
+
+export function PostInfo({ post = {} }) {
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{post.title}</h3>
+
+        <p>
+          {' Posted by  '}
+
+          <UserInfo user={post.user} />
+        </p>
+      </div>
+
+      <p className="PostInfo__body">{post.body}</p>
+
+      {Object.keys(post.comments).length === 0 ? (
+        <b data-cy="NoCommentsMessage">No comments yet</b>
```

Comment:
Looks like you forget to add `<hr />` tag here. 
![image](https://github.com/user-attachments/assets/bfb4e62e-ce53-46b8-a683-a5b3c4cc9a9f)

Reply:
> Looks like you forget to add `<hr />` tag here. ![image](https://private-user-images.githubusercontent.com/120860193/368724818-bfb4e62e-ce53-46b8-a683-a5b3c4cc9a9f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjY3MzAwNzIsIm5iZiI6MTcyNjcyOTc3MiwicGF0aCI6Ii8xMjA4NjAxOTMvMzY4NzI0ODE4LWJmYjRlNjJlLWNlNTMtNDZiOC1hNjgzLWE1YjNjNGNjOWE5Zi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTE5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkxOVQwNzA5MzJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wODAzYWI4NzczYmUzYjVmYmFjZWRkOGI5YjZkNjkyMTUyNWY5MzY0Y2JiMTFkOGEzMTVkMzhjZTBmZDJmNzRiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.JCuwzXKSkyUpdToUzBs0ewPJweQPPd_xp9UcYH1LHoU)

Good point